guides/source/engines.md

```diff
@@ -402,7 +402,7 @@ module Blorgh
 end
 ```
 
-NOTE: The `ApplicationController` class being inherited from here is the
+NOTE: The `ArticlesController` class inherits from
 `Blorgh::ApplicationController`, not an application's `ApplicationController`.
```

Comment:
I know this isn't part of your changeset, but can you change the `an` to `the`, here?